doozer/doozerlib/backend/konflux_client.py

```diff
@@ -296,9 +296,8 @@ def _new_pipelinerun_for_image_build(generate_name: str, namespace: Optional[str
         if additional_tags is None:
             additional_tags = []
         https_url = art_util.convert_remote_git_to_https(git_url)
-        # TODO: In the future the PipelineRun template should be loaded from a remote git repo.
-        template_content = files("doozerlib").joinpath("backend").joinpath("konflux_image_build_pipelinerun.yaml").read_text()
-        template = jinja2.Template(template_content, autoescape=True)
+        response = requests.get(constants.KONFLUX_PlR_TEMPLATE_URL)
```

Comment:
We don't have to pull the template for each PipelineRun. Better to cache it on start.

Reply:
Thanks - updated and retested. Works as expected